src/pixie/fonts.nim

```diff
@@ -72,6 +75,22 @@ proc lineHeight*(typeface: Typeface): float32 {.inline.} =
   ## The default line height in font units.
   typeface.ascent - typeface.descent + typeface.lineGap
 
+proc underlinePosition(typeface: Typeface): float32 {.inline.} =
+  if typeface.opentype != nil:
```

Comment:
No SVG support?

Reply:
No svg font support, if you want to work on svg fonts thats all good but I'm not super interested in them since they have no future.